### DISCUSSION TITLE
Questions about the example .ccd coil file

### DISCUSSION THREAD

**Discussion by windyjunfeng** [simnibs/simnibs#359] [8. Simulation workflows] [answered]
Hi, I have two questions about a commonly used .ccd coil file, Magstim_70mm_Fig8.ccd. I find the z coordinates of coil dipoles between -4.1666 and -8.8333.

Q1: The minimal absolute value of z coordinate should be close to 0. Now it's 4, which means the nearest coil dipole to XY plane in coil coordinate space is 4mm. But we also need to set the distance from coil to scalp when calculating tms efield. If we set the distance parameter as 4, the real nearest distance from coil wire to scalp is 8mm? Is that reasonable?

Q2: I see the instruction file about the coil orientation. The z positive direction is pointing away from the head. But the z coordinates of coil dipoles is negative. According to the definition, the coil is under the scalp if the distance parameter is set lower than 8. It's unreasonable. Actually, I view the corresponding codes and visualize and find the z positive direction of the coil is pointing inward in the head.

Look forward to your reply. Thank you very much! 
![image](https://github.com/simnibs/simnibs/assets/58356387/def0373b-34c4-4253-ad87-4c64e6d5dc85)

**Comment by TorgeW** [comment, marked as answer]
I think you misunderstand the SimNIBS coil orientation. I hope this picture makes it more clear.
![image](https://github.com/simnibs/simnibs/assets/122801447/6aecddb6-bf62-4dca-84aa-b84a25f4bdaa)

Positive z direction is towards the head. So the dipoles in the negative range are inside the coil.
The dipoles are optimized to create the same field as the original coil, the positions of the dipoles are arbitrary. If you want to know the wire to scalp distance, you would need to acquire the coil windings from another source. 
So when you set the coil distance to 0, that is just the same as holding the coil right at the head in the real world. 
The distance for fig of 8 coils is always measured as the coil casing to scalp distance.

**Reply by windyjunfeng** [reply]
I got it. Thank you!
